skt/executable.py

```diff
@@ -966,8 +972,13 @@ def load_config(args):
             cfg['runner'][1]['blacklist']
         )
 
-    # Preparation for output directory option
-    if os.access(cfg.get('workdir'), os.W_OK | os.X_OK):
+    # Create and get an absolute path for the output directory
+    if cfg.get('output_dir'):
+        try:
+            os.mkdir(full_path(cfg.get('output_dir')))
```

Comment:
Can you assign the full path to the output directory first and work with that? That way we don't need to convert it with every use.